<?xml version="1.0"?>

<launch>
<param name="use_sim_time" value="false"/>
  <!-- <group ns="$(arg drone_name)"> -->
    <node name="filter_platform_data" pkg="training_q_learning" type="vicon_filter_platform_data.py" output="screen" >
    </node> 
  <!-- </group>   -->
</launch>
